docs/pages/2020_BIGMAP/sections/qe.rst

```diff
@@ -2,3 +2,349 @@
+    #. Run a self-consistent field calculation on the refined structure.
+    #. Run a band structure calculation at fixed Kohn-Sham potential along a standard path between high-symmetry k-points determined by SeeK-path_.
+
+The workflow uses the PBE exchange-correlation functional with suitable pseudopotentials and energy cutoffs from the `SSSP library version 1.1 <https://www.materialscloud.org/discover/sssp/table/efficiency>`_.
+
+.. K-point mesh is selected to have a minimum k-point density of 0.2 Å-1
+   A Marzari-Vanderbilt smearing of 0.02 Ry is used for the electronic occupations
+
+The workflow should take ~10 minutes on your virtual machine.
+You may notice that ``verdi process list`` now shows more than one entry:
+
+.. code-block:: console
+
+    $ verdi process list
+      PK  Created    Process label             Process State    Process status
+    ----  ---------  ------------------------  ---------------  --------------------------------------
+     218  6s ago     PwBandStructureWorkChain  ⏵ Waiting        Waiting for child processes: 233
+     233  4s ago     PwBandsWorkChain          ⏵ Waiting        Waiting for child processes: 235
+     235  3s ago     PwRelaxWorkChain          ⏵ Waiting        Waiting for child processes: 238
+     238  2s ago     PwBaseWorkChain           ⏵ Waiting        Waiting for child processes: 244
+     244  1s ago     PwCalculation             ⏵ Waiting        Monitoring scheduler: job state QUEUED
+
+    Total results: 5
+
+    Info: last time an entry changed state: 0s ago (at 12:45:40 on 2020-11-25)
+
+While you wait for the workflow to complete, let's start exploring its provenance.
+
+The full provenance graph obtained from ``verdi node graph generate`` will already be rather complex (you can try!), so let's try browsing the provenance interactively instead.
+
+Start the AiiDA REST API:
+
+.. code-block:: console
+
+    $ verdi restapi
+
+and open the |provenance browser| (from the browser inside the virtual machine).
+
+.. |provenance browser| raw:: html
+
+    <a href="https://www.materialscloud.org/explore/ownrestapi?base_url=http://127.0.0.1:5000/api/v4" target="_blank">Materials Cloud provenance browser</a>
+
+.. note::
+
+    The provenance browser is a Javascript application that connects to the AiiDA REST API.
+    Your data never leaves your computer.
+
+.. some general comment on importance of the graph?
+.. a sentence on how to continue from here
+
+Browse your AiiDA database:
+
+    * Start by finding your `Quantum ESPRESSO`_ calculation (the type of node is called a ``CalcJobNode`` in AiiDA, since it is run as a job on a scheduler).
+    * Select ``Calculations`` in the left menu to filter for calculations only.
+    * Inspect the raw inputs and outputs of the calculation, and use the provenance browser to explore the input and output nodes of the calculation and the whole provenance of your simulations.
```

Comment:
Maybe we could provide a more guided thing here? "Start by finding your `Quantum ESPRESSO`_ calculation" can be very non-trivial if you are not familiar with the interface of the explorer. Do you want me to expand a bit this section?

Reply:
Let's hold off on that until I've tested if/how we can show the RestAPI when running the tutorial on the AiiDAlab cluster. I'll open an issue so we don't forget.

Reply:
See #279